source/codegen/metadata/nixnet/functions.py

```diff
@@ -390,8 +390,8 @@
         'returns': 'nxStatus_t'
     },
```

Comment:
What manual changes? We have been keeping the two in sync which obviously doesn't allow for manual changes

Reply:
Ah, I should've probably been more verbose on that commit message. There were some changes to metadata that weren't backported from here to scrapigen earlier. We are going to do that soon, but for now I had to manually make those changes after copying the export from scrapigen.

Reply:
That makes sense, yeah I saw that commit come through my email and it was a red flag